wallet/src/actors/app/handlers/create_vtt.rs

```diff
@@ -42,6 +43,7 @@ pub struct CreateVttRequest {
     fee_type: Option<FeeType>,
     #[serde(default)]
     utxo_strategy: UtxoSelectionStrategy,
+    selected_utxos: Vec<model::OutPtr>,
```

Comment:
To make this field optional and default to `[]`:

```suggestion
    #[serde(default)]
    selected_utxos: Vec<model::OutPtr>,
```

Reply:
Also, you can change it into a `HashSet`, so it has the same type everywhere.